.npmrc

```diff
@@ -1,3 +0,0 @@
-auto-install-peers = true
-lockfile = true
-link-workspace-packages = true
```

Comment:
I don't think we should remove the file, since [pnpm is using the `.npmrc` file as well](https://pnpm.io/settings).